src/main/java/xyz/holocons/mc/holoitemsrevamp/enchantment/SSKSword.java

```diff
@@ -0,0 +1,91 @@
+package xyz.holocons.mc.holoitemsrevamp.enchantment;
+
+import com.destroystokyo.paper.MaterialTags;
+import com.strangeone101.holoitemsapi.enchantment.CustomEnchantment;
+import com.strangeone101.holoitemsapi.enchantment.EnchantmentAbility;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.format.TextDecoration;
+import org.bukkit.Location;
+import org.bukkit.Particle;
+import org.bukkit.Sound;
+import org.bukkit.World;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+import org.jetbrains.annotations.NotNull;
+import xyz.holocons.mc.holoitemsrevamp.HoloItemsRevamp;
+import xyz.holocons.mc.holoitemsrevamp.Util;
+
+import java.util.List;
+
+public class SSKSword extends CustomEnchantment implements EnchantmentAbility {
+    private final HoloItemsRevamp plugin;
+
+    public SSKSword(HoloItemsRevamp plugin) {
+        super(plugin, "ssk_sword");
+        this.plugin = plugin;
+    }
+
+    @Override
+    public int getMaxLevel() {
+        return 1;
+    }
+
+    @Override
+    public boolean conflictsWith(@NotNull Enchantment other) {
+        return false;
+    }
+
+    @Override
+    public boolean canEnchantItem(@NotNull ItemStack itemStack) {
+        return MaterialTags.AXES.isTagged(itemStack);
+    }
+
+    @Override
+    public @NotNull Component displayName(int level) {
+        return Component.text()
+            .color(NamedTextColor.GRAY)
+            .decoration(TextDecoration.ITALIC, false)
+            .append(Component.text("SSK Sword"))
+            .build();
+    }
+
+    @Override
+    public int getCostMultiplier() {
+        return 12;
+    }
+
+    @Override
+    public void onPlayerAttack(EntityDamageByEntityEvent event, ItemStack itemStack) {
+        if(!(event.getEntity() instanceof LivingEntity target)) {
+            return;
+        }
+
+        // This is what Klin used, and it seems to be equivalent to damage after-sharpness after-plugins before-prot4
+        final var amountToHeal = event.getDamage();
+        Util.healEntity(target, amountToHeal);
+        event.setDamage(0);
```

Comment:
I think you could just set the damage to negative to heal. Gotta test that.

Reply:
Yep, it works. I'm gonna keep healEntity in Util and port it to DAura at some point.